include/grpcpp/security/server_credentials.h

```diff
@@ -96,7 +96,7 @@ class ServerCredentials : private grpc::internal::GrpcLibrary {
   // insecure builds are removed from gRPC.
   virtual bool IsInsecure() const { return false; }
```

Comment:
I think we can remove this method now.  You've already removed one of the two call sites (xds_server_credentials.cc), and I think you can make the same change in the other one (xds_credentials.cc).